src/main/java/io/r2dbc/pool/ConnectionPoolConfiguration.java

```diff
@@ -151,6 +165,10 @@ PoolMetricsRecorder getMetricsRecorder() {
 
         private PoolMetricsRecorder metricsRecorder = new SimplePoolMetricsRecorder();
 
+        private boolean registerJmx = true;
```

Comment:
Shouldn't this be `false`?

Reply:
Ah, I pushed my update that makes default `jmx=false` to wrong place. Let me also address the `name=null` and update the PR later.

Reply:
updated the PR.